WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalErrorRetryTableViewCell.swift

```diff
@@ -0,0 +1,142 @@
+        static let padding          = CGFloat(4)
+    }
+
+    /// One of the arranged subviews : a "refresh" Gridicon
+    private let retryImageView: UIImageView
+
+    /// One of the arranged subviews : user-facing text
+    private let retryLabel: UILabel
+
+    // MARK: VerticalErrorRetryTableViewCellAccessoryView
+
+    init() {
+        self.retryImageView = {
+            let dismissImage = Gridicon.iconOfType(.refresh).imageWithTintColor(WPStyleGuide.wordPressBlue())
+            let imageView = UIImageView(image: dismissImage)
+
+            imageView.translatesAutoresizingMaskIntoConstraints = false
+            imageView.contentMode = .scaleAspectFit
+
+            return imageView
+        }()
+
+        self.retryLabel = {
+            let label = UILabel()
+
+            label.translatesAutoresizingMaskIntoConstraints = false
+
+            label.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .bold)
+            label.textColor = WPStyleGuide.wordPressBlue()
+            label.textAlignment = .center
+
+            label.text = NSLocalizedString("Retry", comment: "Title for accessory view in the empty state table view cell in the Verticals step of Enhanced Site Creation")
+            label.sizeToFit()
+
+            return label
+        }()
+
+        super.init(frame: .zero)
+
+        initialize()
+    }
+
+    // MARK: UIView
+
+    required init(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: Private behavior
+
+    private func initialize() {
+        translatesAutoresizingMaskIntoConstraints = false
+
+        axis = .horizontal
+        alignment = .center
+        spacing = Parameters.padding
+
+        addArrangedSubviews([retryImageView, retryLabel])
+
+        NSLayoutConstraint.activate([
+            heightAnchor.constraint(greaterThanOrEqualToConstant: Parameters.minimumHeight),
+            retryImageView.widthAnchor.constraint(equalToConstant: Parameters.retryDimension),
+            retryImageView.heightAnchor.constraint(equalToConstant: Parameters.retryDimension),
+        ])
+    }
+}
+
+// MARK: - VerticalErrorRetryTableViewCell
+
+/// Responsible for apprising the user of an error that occurred, accompanied by a visual affordance to retry the
+/// preceding action.
+///
+final class VerticalErrorRetryTableViewCell: UITableViewCell, ReusableCell {
+
+    // MARK: Properties
+
+    /// A collection of parameters uses for view layout
+    private struct Parameters {
+        static let height           = CGFloat(44)
+        static let trailingInset    = CGFloat(16)
+    }
+
+    /// A subview akin to an accessory view
+    private let retryAccessoryView = VerticalErrorRetryTableViewCellAccessoryView()
+
+    // MARK: UITableViewCell
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: .default, reuseIdentifier: VerticalErrorRetryTableViewCell.cellReuseIdentifier())
```

Comment:
Shouldn't this be 
```suggestion
        super.init(style: style, reuseIdentifier: VerticalErrorRetryTableViewCell.cellReuseIdentifier())
```
or is it because you want to force .default?

Reply:
Correct! The default style vends a `Simple cell with text label and optional image view`.

Reply:
👍